inst/schema/Response.schema.json

```diff
@@ -8,7 +8,7 @@
           "const": "success"
         },
         "data": {
-          "oneOf": [
+          "anyOf": [
```

Comment:
Is `oneOf` not more appropriate here as we want to match exactly one?

* `anyOf`:

> An instance validates successfully against this keyword if it validates successfully against at least one schema defined by this keyword's value.

* `oneOf`:

> An instance validates successfully against this keyword if it validates successfully against exactly one schema defined by this keyword's value.

https://tools.ietf.org/id/draft-handrews-json-schema-validation-01.html#rfc.section.6.7.2